comit/src/expiries.rs

```diff
@@ -899,8 +908,8 @@ mod tests {
     fn dev_net_herc20_hbit_expiries() {
         let (a, b) = expiry_offsets_herc20_hbit(Network::Dev);
 
-        assert_eq!(a, 44.seconds().into());
-        assert_eq!(b, 35.seconds().into());
+        assert_eq!(a, 218.seconds().into());
+        assert_eq!(b, 155.seconds().into());
```

Comment:
This is not good. Why did these values go up that much? If we ever want to test refund in our e2e tests, this is amount of time we would have to wait. 218 seconds is almost 4 minutes of just waiting.

Can we somehow keep these numbers under 1 minute?

Reply:
> This is not good. Why did these values go up that much? If we ever want to test refund in our e2e tests, this is amount of time we would have to wait. 218 seconds is almost 4 minutes of just waiting.
> 
> Can we somehow keep these numbers under 1 minute?

Yes we can with separate mainnet and dev value. Will do.

Reply:
Cool, thank you!